### PR TITLE
Fix CPU-Offload: Send groups of parameter lists as the FP16 parameters

### PR DESCRIPTION
**Pull request by RezaYazdaniAminabadi**
This fixes https://github.com/microsoft/DeepSpeed/issues/1751

### CODE REVIEW

File: deepspeed/runtime/zero/stage_1_and_2.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def __init__(self,
             see_memory_usage(f"Before moving param group {i} to CPU")
             # move all the parameters to cpu to free up GPU space for creating flat buffer
             move_to_cpu(self.bit16_groups[i])
-            see_memory_usage(f"After moving param group {i} to CPU", force=False)
+            see_memory_usage(f"After moving param group {i} to CPU", force=True)
```
Comment (tjruwase) [inline deepspeed/runtime/zero/stage_1_and_2.py:286]: Please revert this change.

**Review by tjruwase — APPROVED** [review]

File: deepspeed/runtime/zero/stage_1_and_2.py (hunk 1 above)
Comment (RezaYazdaniAminabadi) [reply deepspeed/runtime/zero/stage_1_and_2.py]: done!
